style/typography.js

```diff
@@ -49,8 +49,8 @@ const H3 = styled.h3`
 
 const P = styled.p`
   font-family: ${fonts.poppins};
-  font-weight: ${fontWeights.regular};
-  font-size: 1em;
+  font-weight: ${fontWeights.normal};
+  font-size: clamp(1em, 2vw, 1.2em);
```

Comment:
I think we should keep the font-size to be 1em - on my desktop, the clamp is making the font-size be 1.2em and it looks too big (and is also a different font size than the featured story description at the top).

Reply:
^^ will also help in reducing the ellipses / newsletter sign up part to be smaller and more similar in size to hi-fi